tests/regression/roles/sonic_system/defaults/main.yml

```diff
@@ -12,7 +12,8 @@ tests:
         ipv4: false
         ipv6: false
       auto_breakout: ENABLE
-      load_share_hash_algo: JENKINS_HASH_HI
+      # Use only on switch models that support this.
+      #load_share_hash_algo: JENKINS_HASH_HI
```

Comment:
"load_share_hash_algo"'s are commented out. Are those intended?

Reply:
Yes. This test is only supported on a Z9664f switch. It is commented out instead of deleted so that we can still run it on a Z9664f by uncommenting the indicated lines of the test.